Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h

```diff
@@ -60,7 +60,7 @@ class CTPPSPixelSimTopology : public CTPPSPixelTopology
 
       // rows (x segmentation)
       if ( arow == 0 ) {
-        lower_x = 0;
+        lower_x = 0.05;
```

Comment:
This file is filled with magic numbers. They should be replaced with constants now or in a later pull request. See [7.6](https://github.com/cms-sw/cms-sw.github.io/blob/3e89200bde531a9aa1d01d4c92c1d65a5f6c87fd/cms_coding_rules.md#7----design-and-coding-guidelines-1) of our coding guidelines.

Reply:
done